keras_cv/utils/visualize.py

```diff
@@ -0,0 +1,67 @@
+import tensorflow as tf
+import PIL.ImageColor as ImageColor
+
+
+def draw_masks_on_images(image, mask, color="red", alpha=0.4):
+    """Draws masks on images.
+
+    Args:
+        image: an uint8 tensor with shape (N, img_height, img_height, 3)
+        mask: an uint8 tensor of shape (N, img_height, img_height) with values
+            between either 0 or 1.
+        color: color to draw the keypoints with. Default is red.
+        alpha: transparency value between 0 and 1. (default: 0.4)
+    Returns:
+        Masks overlaid on images.
+
+    Raises:
+        ValueError: On incorrect data type for images or masks.
+    """
+
+    def _blend(image1, image2, factor):
+        difference = image2 - image1
+        scaled = factor * difference
+        # Do addition in float.
+        blended = image1 + scaled
+
+        # Interpolate
+        if factor > 0.0 and factor < 1.0:
+            # Interpolation means we always stay within 0 and 255.
+            return tf.round(blended)
+        return blended
+
+    if image.dtype != tf.uint8:
```

Comment:
can we support all integer types?

Reply:
@LukeWood 
Function was working fine with every int type except int8. 

Reason.
`tf.int8` cannot be used to represent 0, 255 value range.


 when I casted float32 0-255 image to int8 
it represented value above 127 with signed equivalent i.e -x.

Reply:
all integer dtype will work except int8
as int8 range is -127, 127.

Reply:
lets remove this assertion then - right?

Reply:
ah - it was updated.  thanks